requirements.in

```diff
@@ -8,3 +8,4 @@ pycodestyle==2.4.0
 mock==2.0.0
 path.py==8.2.1
 django-crum==0.7.3
+django-fake-model==0.1.4
```

Comment:
This is only for testing right? Is this fake-model something we should refactor into other tests in the future?

Reply:
yes, this is only for testing, probably in other tests where we need a model from the platform